runtime/interface.go

```diff
@@ -36,9 +37,13 @@ type Block struct {
 	Timestamp int64
 }
 
+type ResolvedLocation = sema.ResolvedLocation
+
 type Interface interface {
-	// ResolveImport resolves an import of a program.
-	ResolveImport(Location) ([]byte, error)
+	// ResolveLocation resolves an import location.
+	ResolveLocation(identifiers []Identifier, location Location) []ResolvedLocation
+	// GetCode returns the code at a given location
+	GetCode(location Location) ([]byte, error)
```

Comment:
Split the single import function into two function: resolving the import location, and getting the code for a resolved location